src/main/java/Todo.java

```diff
@@ -0,0 +1,17 @@
+public class Todo extends Task {
+    private String name;
+    private boolean done;
```

Comment:
Should not recreate a new private variable in Todo. Instead use Task's protected isDone and name